webplugin/js/app/kommunicate-ui.js

```
@@ -139,15 +139,35 @@ KommunicateUI = {
             type: 'GET',
             global: false,
             success: function (result) {
+                const { title, description } = result.data;
+                // Check if the title contains Cloudflare or security challenge related words
+                if (
+                    title &&
+                    title.includes('Attention Required!')
+                ) {
+                    throw Error("Cloudflare or security block detected. No preview available.") // Skip rendering the preview or return a default message
+                }
+
+                // Check for other possible issues with the preview data (e.g., missing meta information)
+                if (!title || !description) {
+                    throw Error("Missing metadata for preview. No preview available.")
```

do we really need to throw the error, or can we just return?